src/services/authService.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LoginInterface,
+  LoginResponseInterface,
+  RegisterResponseInterface,
+} from "../types/authTypes";
+import { anonymPostRequest } from "./apiService";
+
+export async function registerRequest(config: LoginInterface) {
+  return anonymPostRequest(
+    "/register",
+    config
+  ) as Promise<RegisterResponseInterface>;
+}
+
+export async function loginRequest(config: LoginInterface) {
+  return anonymPostRequest("/login", config) as Promise<LoginResponseInterface>;
+}
+
+export async function authRequest(token: string) {
```

Comment:
`https://reqres.in/ ` provide only authToken by login request, but doesn’t give any auth endpoint, so I'll hardcode this functionality and return userId